modules/vba_documents/app/swagger/vba_documents/v0/controller_swagger.rb

```diff
@@ -137,6 +137,47 @@ class ControllerSwagger
         end
       end
 
+      swagger_path '/uploads/{id}/download' do
+        operation :get do
+          key :summary, 'Download zip of "what the server sees"'
```

Comment:
If we are going to limit this in production then we should include it in the documentation.

Reply:
resolved in a7df32a